src/lib/api/serverFetch.ts

```diff
@@ -9,3 +14,17 @@ export async function performServerFetch(
         return Promise.resolve(await result.text());
     } else throw result;
 }
+
+export async function performServerFetch<T>(
+    input: string | Request | URL,
+    init?: RequestInit | undefined,
+): Promise<ApiResponseWrapper<T>> {
+    try {
+        const result = await fetch(input, init);
```

Comment:
I'd prefer it if we renamed "result" to "response".

Reply:
Agreed